tests/test_session.py

```diff
@@ -213,7 +213,6 @@ def test_start_process(demo_session):
     with pytest.raises(FigoException):
         demo_session.start_process(process_token)
 
-
```

Comment:
`flake8` will choke on that, it demands two blank lines between global function definitions.